### ISSUE TITLE
primeng accessibilty violations

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#274]
### Describe the bug

The p-select, p-radiobutton, p-iconField, p-menubar, p-progressbar component has an accessibility violation when running the IBM Equal Accessibility Checker: 

1. Form control element <input> has no associated label - [for p-radiobutton]
2. Form control element <input> has no associated label - [for p-password]
3. Form control with "searchbox" role has no associated label
4. Form control element <input> has no associated label - [for p-select]
5. Form control with "progressbar" role has no associated label
6. Table has no headers identified  - [for p-treetable]
7. Complex table does not have headers for each cell properly defined with 'header' or 'scope' - [for p-treetable]
8. The 'id' "pn_id_1172_content" specified for the ARIA property 'aria-controls' value is not valid
9. The ARIA attributes "aria-level" are not valid for the element <li> with ARIA role "menuitem" - [for p-menubar]
10. Multiple elements with "form" landmarks within the same parent region are not distinguished from one another because they have the same "" label

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz-starters-sfmwlzcw.stackblitz.io

### Environment

stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the error:
    - Form control element <input> has no associated label - [for p-radiobutton]
    - Form control element <input> has no associated label - [for p-password]
    - Form control with "searchbox" role has no associated label
    - Form control element <input> has no associated label - [for p-select]
    - Form control with "progressbar" role has no associated label
    - Table has no headers identified  - [for p-treetable]
    - Complex table does not have headers for each cell properly defined with 'header' or 'scope' - [for p-treetable]
    - The 'id' "pn_id_1172_content" specified for the ARIA property 'aria-controls' value is not valid
    - The ARIA attributes "aria-level" are not valid for the element <li> with ARIA role "menuitem" - [for p-menubar]
    - Multiple elements with "form" landmarks within the same parent region are not distinguished from one another because they have the same "" label

### Expected behavior

No accessibility violations are present for the p-dialog component

> Migrated from `primefaces/primeng#18942` — originally by `@srujana10499` on 2025-09-26

**Comment by geromegrignon** [comment]
*[Originally posted by `@JohnFTitor` on 2025-10-27T12:50:12.000Z]*

The specific issue for the menuitem is the aria-level: The ARIA attributes "aria-level" are not valid for the element `<li>` with ARIA role "menuitem"
Just in case someone picks this up

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-04-26T00:38:24.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
